portfolio/src/main/webapp/script.js

```diff
@@ -100,4 +100,67 @@ function submitComment() {
 function likeComment(commentId) {
   const request = new Request(`/like?commentId=${commentId}`, { method: 'POST' })
   fetch(request).then(() => getComments());
+}
+
+function checkLoginStatus() {
```

Comment:
Name should be descriptive of what the method actually does. Maybe something like `addLoginStatus`.

Reply:
👌